### DISCUSSION TITLE
How to install Firefly III exchange rates collector

### DISCUSSION THREAD

**Discussion by grzbrum** [firefly-iii/firefly-iii#8860] [Firefly III]
Hi,
lots of questions, but don't want to do anything wrong:
Which files must be copied to which directories?
Where must the API key be entered?
Is this done automatically with the CRON job or which file do I have to execute?

**Comment by JC5** [comment]
There is nothing yet in Firefly III that uses the exchange rates, but if you set `ENABLE_EXCHANGE_RATES=true` the cron job will start downloading them automatically.

**Comment by grzbrum** [comment]
Ok. Thank you.
